is/esb/newmount.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+# $1 = Azure storage account name 
+# $2 = Azure storage account key 
+# $3 = Azure file share name 
+# $4 = Mountpoint path (required for multi-instance VMSS, else set to empty string) 
+
+sudo bash -c 'echo "fs.file-max = 1048576" >> /etc/sysctl.conf'
+sudo sysctl -p
+
+if [ -d "/data/boomi" ] && [ -d "/data/export" ] && [ -d "/data/files" ] 
+then
+  echo "good"
+else
+    sudo mkdir -p /data/boomi && sudo mkdir /data/files && sudo mkdir /data/export && sudo chmod 777 /data && sudo chmod 777 /data/boomi && sudo chmod 777 /data/files && sudo chmod 777 /data/export
+fi
+
+#if [ ! -d "/etc/smbcredentials" ]; then
+#sudo mkdir -p /etc/smbcredentials
+#fi
+#if [ ! -f "/etc/smbcredentials/$1.cred" ]; then
+    #sudo bash -c 'echo "username='$1'" >> /etc/smbcredentials/'$1'.cred'
+    #sudo bash -c 'echo "password='$2'" >> /etc/smbcredentials/'$1'.cred'
+#fi
+
+#sudo chmod 600 /etc/smbcredentials/$1.cred
+
+sudo apt-get -y update
+sudo apt-get -y install nfs-common
+sudo apt install nfs-kernel-server
```

Comment:
Line 29 is using `apt` not `apt-get` - presumably it should be `apt-get`?

Reply:
Thanks TC, i have made the changes